### DISCUSSION TITLE
Fullscreen option and background change.

### DISCUSSION THREAD

**Discussion by sgrxrx** [xgi/houdoku#125] [Feature Requests]
It would be nice to have a true full screen option. Clicking F11 does make it full screen but the setting bar still remains so it would be nice to have an option that can hide setting bar and give fullscreen for pages. And yeah changeable background would be good to like a black one.

**Comment by xgi** [comment]
I actually implemented this fullscreen behavior a few days ago with #124, it'll be included in the next release later this week.

**Reply by sgrxrx** [reply]
ohh thanks! looking forward to the next release.
